Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "openidconnect"
-version = "2.0.0-alpha.3"
+version = "2.0.0-alpha.4"
```

Comment:
2.0.0-alpha.3 still hasn't been released, so that's the correct version here I think

Reply:
Done.